ui/component/channelSelector/view.jsx

```diff
@@ -99,12 +111,19 @@ export default function ChannelSelector(props: Props) {
                 isSelected
                 claimsByUri={claimsByUri}
                 doFetchUserMemberships={doFetchUserMemberships}
+                isPublishMenu={isPublishMenu}
+                isTabHeader={isTabHeader}
```

Comment:
- No more `isPublishMenu` and `isTabHeader` -- 2 instances
- Also remove from `ListItemProps`

Reply:
They're not exactly the same. They use different classes.

Reply:
hm ... there are 2 instance of `ChannelListItem` where `isPublishMenu` and `isTabHeader` is being passed in, but seems like `ChannelListItem/ListItemProps` no longer needs them.

But ok to leave if coming back in the future.